docs/Child-friendly-EmulationStation.md

```diff
@@ -13,23 +13,23 @@ These modes allow administrators to lock down (part of) the system, so that naiv
 ***
 
 # UI modes
-To allow different levels of system protection, user modes were introduced with different sets of functionalities. These modes are Full, Kiosk and Kid. Generally speaking, kids do not need to edit system settings, nor will visiting friends. The full set of options also complicates the UI, and the kid and kiosk modes hide these when they are not needed.
+To allow different levels of system protection, user modes have been introduced to allow the user to restrict certain settings and files from use. The three new modes are Full, Kiosk and Kid. In general, visiting friends and children do not need to modify system settings. The full set of options also complicates the UI, and the kid and kiosk modes hide these for a simpler experience.
 
 Setting UI mode to Full will give you the default ES user interface. This includes options such as CONFIGURE INPUT, UI SETTINGS, SCRAPER, but also EDIT THIS GAMES METADATA, and many other menu items that allow you to configure the system.
 
-Kiosk and Kid mode expose fewer of the menu options of Emulation station, only showing menu items that do not affect the system configuration (i.e. do not survive a reboot), with the exception of setting favorites. In Kiosk Mode, the user is still able to toggle the favorite status of a game. In Kid-mode, even this functionality is removed.
+Kiosk and Kid mode expose fewer of the menu options of Emulation Station, only showing menu items that do not affect the system configuration (i.e. do not survive a reboot). In Kiosk Mode, the user is still able to toggle the favorite status of a game. In Kid-mode, this functionality is removed.
 
-In addition to this, the Kiosk and Kids modes allow for filtering of the game lists to remove unwanted items from the view, without actually removing the files from the system. The Kiosk mode only shows items in the game lists that are not explicitly set to be hidden (aka blacklisting). The Kid only shows items which are explicitly selected to be kid-friendly (aka whitelisting). Items which are set to be both `hidden` and `kidgame`, will be visible while in UI mode `kid`, but not while in UI mode `Kiosk`.
+In addition to this, the Kiosk and Kid modes allow for filtering of the game lists to hide unwanted games from view. This does not remove any files from the system. Kiosk mode only shows items in the game lists that are not explicitly set to be hidden (aka blacklisting). The Kid only shows items which are explicitly selected to be kid-friendly (aka whitelisting). Items which are set to be both `hidden` and `kidgame`, will be visible while in UI mode `kid`, but not while in UI mode `Kiosk`.
```

Comment:
_The Kid only_ should perhaps be _The Kid mode only_.

Reply:
Fixed in 1a9b8410db3f8f50cc44aebb5353dbc80f11b19d